src/styles.css

```
@@ -358,15 +358,48 @@
 	}
 
 	.thinkex-thinking-mark {
+		--thinkex-thinking-cycle: 2.2s;
 		overflow: visible;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)